### ISSUE TITLE
Scroll gets stuck when using scrollbar or dragging touch

### ISSUE THREAD

**Issue by IndiePix** [flet-dev/flet#3963]
### Duplicate Check

- [x] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

When running the app in standalone mode, the scrollbar gets stuck and unresponsive (it makes little jumps each time you drag it). The same happens when you touch scroll by swiping up or down, it moves an step and stops abruptly.

Note 1: Scrolling works fine and smoothly in WEB BROWSER mode, it only happens when running as an standalone app.
Note 2: Scrolling with the mousewheel also works fine.

### Code sample

<details open><summary>Code</summary>

```python
import flet as ft

def main(page: ft.Page):
    page.title = "GridView Example"
    page.theme_mode = ft.ThemeMode.DARK
    page.padding = 50
    page.update()

    images = ft.GridView(
        expand=1,
        runs_count=5,
        max_extent=150,
        child_aspect_ratio=1.0,
        spacing=5,
        run_spacing=5,
    )

    page.add(images)

    for i in range(0, 60):
        images.controls.append(
            ft.Image(
                src=f"https://picsum.photos/150/150?{i}",
                fit=ft.ImageFit.NONE,
                repeat=ft.ImageRepeat.NO_REPEAT,
                border_radius=ft.border_radius.all(10),
            )
        )
    page.update()

ft.app(main)
```

</details>

### To reproduce

1. Run GridView example from Flet Docs in standalone mode 
2. Try to move the scrollbar or drag touch the screen to scroll through the grid
3. Notice how it doesn't scroll smoothly and does little jumps / stucks each time an scroll action is triggered from the scrollbar or touch dragging.

### Expected behavior

Smooth scroll when using the scrollbar or dragging touch (in this case with a touch screen on windows)

### Screenshots / Videos

<details open>
<summary>Screen capture:</summary>

https://github.com/user-attachments/assets/c26519b1-d525-405b-8ca1-6c2b917ae263

</details>

### Operating System

Windows

### Operating system details

Windows 11

### Flet version

0.24.1

### Regression

Yes, it used to work in a previous Flet version (please specify the version in additional details)

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Additional details

In 0.23 the scrollbar worked smoothly but the touch dragging still had this bug.
In 0.24 neither of the two works.

**Comment by Siva-Rajan-R** [comment]
Yes bro same thing happens to me

**Comment by HenryNomeland** [comment]
I am also getting this issue - would be nice to have a fix

**Comment by ndonkoHenri** [comment]
@Siva-Rajan-R @HenryNomeland provide specifications (OS, Flet version etc).

**Comment by YagamiKAI** [comment]
this is true I am on Windows 11 64 bits I code with the latest versions python, flet, I can add to this recent bug the fact of not being able to use the mouse in horizontal scroll

**Comment by YagamiKAI** [comment]
adding to that thank you in advance for your incredible library

**Comment by CDXC-55uh** [comment]
**Comment**
The same for me. 

_Android_: On Android, in the Flet app, it works smoothly.

_Windows 11_: On Windows 11 in the desktop version, scrolling jumps only once by a small amount when moving up or down. The same thing happens when I try to scroll with my finger on the touch screen.

**Operating System**
Windows

**Operating system details**
Windows 11

**Flet version**
0.24.1

**Comment by Siva-Rajan-R** [comment]
@HenryNomeland windows os flet 0.24.1

**Comment by DavidFraifer** [comment]
Same for me.

While dragging it stops, and only jums when you click other place of the scroll.

https://github.com/user-attachments/assets/136dafb4-5631-495d-8eed-c07354e15417


Operating system details
Windows 11

Flet version
0.24.1

**Comment by OwenMcDonnell** [comment]
[PR here](https://github.com/flet-dev/flet/pull/4145)
